lib/datadog/core/configuration/settings.rb

```diff
@@ -618,6 +619,19 @@ def initialize(*_)
           o.default { ENV.fetch(Core::Environment::Ext::ENV_VERSION, nil) }
           o.lazy
         end
+
+        # Client-side telemetry configuration
+        # @public_api
+        settings :telemetry do
+          # Enable telemetry collection. This allows telemetry events to be emitted to the telemetry API.
+          #
+          # @default `DD_INSTRUMENTATION_TELEMETRY_ENABLED` environment variable, otherwise `true`
```

Comment:
The changelog says that "Implement telemetry, disable by default", but this seems like telemetry is enabled by default?

Reply:
Hey @gingerlime that's actually correct -- we initially merged it enabled (as you see here) but shortly before release we disabled it in https://github.com/DataDog/dd-trace-rb/pull/2233

Hope this helps :)

Reply:
Thank you. The changelog only links to the original PR so I didn't think to check. Perhaps you want to update it? in any case, it's good to know that it's disabled by default!

Reply:
Ahhh right, oops, you're absolutely right and I've just updated the changelog with the link to the second PR.

Reply:
Thank you @ivoanjo !